pandas/core/groupby/generic.py

```diff
@@ -41,6 +44,10 @@
 
 from pandas.plotting._core import boxplot_frame_groupby
 
+NamedAgg = namedtuple("NamedAgg", ["column", "aggfunc"])
```

Comment:
can you add a comment on what this is (can we add a doc-string)?

Reply:
We could using `typing.NamedTuple`.